core/store/src/db.rs

```diff
@@ -461,8 +461,17 @@ fn rocksdb_options() -> Options {
     opts.set_max_open_files(512);
     opts.set_keep_log_file_num(1);
     opts.set_bytes_per_sync(1048576);
-    opts.set_write_buffer_size(1024 * 1024 * 512 / 2);
-    opts.set_max_bytes_for_level_base(1024 * 1024 * 512 / 2);
+
+    #[cfg(feature = "no_cache")]
```

Comment:
is there a reason why we have to make this distinction? Couldn't we simply lower the write buffer size?

Reply:
You are right, I changed it to always reduce write buffer to 64mb.